components/client/src/test/unit/java/com/hotels/styx/client/origincommands/DisableOriginTest.java

```diff
@@ -1,37 +0,0 @@
-/*
-  Copyright (C) 2013-2018 Expedia Inc.
-
-  Licensed under the Apache License, Version 2.0 (the "License");
-  you may not use this file except in compliance with the License.
-  You may obtain a copy of the License at
-
-  http://www.apache.org/licenses/LICENSE-2.0
-
-  Unless required by applicable law or agreed to in writing, software
-  distributed under the License is distributed on an "AS IS" BASIS,
-  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  See the License for the specific language governing permissions and
-  limitations under the License.
- */
-package com.hotels.styx.client.origincommands;
-
-import com.hotels.styx.api.Id;
-import com.hotels.styx.support.EqualsTester;
-import org.testng.annotations.Test;
-
-import static com.hotels.styx.api.Id.GENERIC_APP;
-import static com.hotels.styx.api.Id.id;
-
-public class DisableOriginTest {
-    @Test
```

Comment:
Why is `DisableOriginTest` and `EnableOriginTest` removed?

What functionality are they actually testing? Looking at code I am still clueless.

Reply:
For some reason they were just testing the `equals` methods, and had two support classes just for these tests.